tests/unit_tests/forward_model_runner/test_job.py

```diff
@@ -141,7 +141,8 @@ def test_memory_profile_in_running_events():
     else:
         for oom_score in emitted_oom_score_values:
             assert oom_score is not None, "No oom_score, are you not on Linux?"
-            assert -1000 <= oom_score <= 1000
+            # Upper limit "should" be 1000, but has been proven to overshoot.
+            assert -1000 <= oom_score <= 1001
```

Comment:
Then why to test for max if that is not reliable?

Reply:
fair enough. Will remove.